doc/src/manual/mathematical-operations.md

```diff
@@ -331,7 +331,18 @@ comparison is undefined. It is strongly recommended not to use expressions with
 as printing) in chained comparisons. If side effects are required, the short-circuit `&&` operator
 should be used explicitly (see [Short-Circuit Evaluation](@ref)).
 
-### Operator Precedence
+### Elementary Functions
+
+Julia provides a comprehensive collection of mathematical functions and operators. These mathematical
+operations are defined over as broad a class of numerical values as permit sensible definitions,
+including integers, floating-point numbers, rationals, and complexes, wherever such definitions
+make sense.
+
+Moreover, these functions (like any Julia function) can be applied in "vectorized" fashion to
+arrays and other collections with the [dot syntax](@ref man-vectorized) `f.(A)`,
+e.g. `sin.(A)` will compute the elementwise sine of each element of an array `A`.
```

Comment:
I am completely ambivalent to this language; like you said, I just moved it.

But I think I agree that the wording you suggested seems better. Should I change it, or make a new cleanup PR?